scripts/features.R

```diff
@@ -0,0 +1,125 @@
+source('scripts/packages.R')
+
+# this script is going to build a table of features from all phase 2 sites
+# there are 3 columns for features in mergin form but only one in the hab con sheet
+# easier to do it this way so we can get only the features and utms, required for hab_features object in `outgoing-mapping.R` script
+
+# import geopackages from mergin and rearrange then burn to csv
+
+# name the project directory we are pulling from
+dir_project <- 'bcfishpass_skeena_20220823'
+
+# there are tables with no new events and the types are mangels so remove dataframes with only 1 feature
+form_names_to_remove <- c('form_fiss_site_202209070609', 'form_fiss_site_202209100809')
+
+# list all the fiss form names in the file
+form_names_l <- list.files(path = paste0('../../gis/mergin/',
+                                         dir_project),
+                           pattern = glob2rx('*site_2022*.gpkg'),
+                           full.names = T
+) %>%
+  stringr::str_subset('form_fiss_site_202209070609', negate = T) %>%
+  stringr::str_subset('form_fiss_site_202209100809', negate = T)
+
+
+
+# read all the forms into a list of dataframes using colwise to guess the column types
+# if we don't try to guess the col types we have issues later with the bind_rows join
+form <- form_names_l %>%
+  purrr::map(sf::st_read) %>%
+  purrr::map(plyr::colwise(type.convert)) %>%
+  # name the data.frames so we can add it later as a "source" column - we use basename to leave the filepath behind
+  purrr::set_names(nm = basename(form_names_l)) %>%
+  bind_rows(.id = 'source')
+
+
+# see the names of our form
+names(form)
+
+# select the columns we want
```

Comment:
tidyr::pivot_longer should put these all in the same columns with one line of code.  I will show you